### PR TITLE
fix(calculated elements): path expressions in `func.args` within `xpr`

### PR DESCRIPTION
**Pull request by patricebender**
the `args` of the `func` have not been considered, now the correct `join` is produced

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Our commit messages use a simplified form of [conventional commits](https://www.
 [body]
 ```
 
-### Commit type prefixes
+### Commit type prefixes≥
```
Comment (patricebender) [inline CONTRIBUTING.md:50]:
```suggestion
### Commit type prefixes
```

File: db-service/test/cqn4sql/calculated-elements.test.js [hunk 2]
```diff
@@ -63,6 +63,16 @@ describe('Unfolding calculated elements in select list', () => {
       }`
     expect(query).to.deep.equal(expected)
   })
+  it('calc elem is function', () => {
```
Comment (patricebender) [inline db-service/test/cqn4sql/calculated-elements.test.js:66]:
```suggestion
  it('calc elem is xpr with multiple functions as args', () => {
```

File: db-service/test/bookshop/db/booksWithExpr.cds [hunk 3]
```diff
@@ -32,10 +32,15 @@ entity Books {
   authorAdrText = author.addressText;
 
   authorAge: Integer = years_between( author.sortCode, author.sortCode );
+  authorAgeNativePG: Integer = DATE_PART('year', author.dateOfDeath) - DATE_PART('year', author.dateOfBirth);
+  
+  // calculated element is `xpr` which has subsequent `xpr`
+  authorAgeInDogYears: Integer = ( DATE_PART('year', author.dateOfDeath) - DATE_PART('year', author.dateOfBirth) ) * 7;
```
Comment (patricebender) [inline db-service/test/bookshop/db/booksWithExpr.cds:38]: mir gehen Ideen für sinnvolle calculated elements aus

Comment (stewsk) [reply db-service/test/bookshop/db/booksWithExpr.cds:38]: :-)

**Review by stewsk — APPROVED** [review]

**Pull request merged**